tools/release/create-pr-to-merge-release-branch-to-main.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -e
+
+# Load common lib.
+CURR_DIR="$(dirname "$0")"
+. "${CURR_DIR}/common.sh"
+
+check_required_tools
+
+# Ensure there are not uncommitted changes.
+if [ -n "$(git status --porcelain=v1 2> /dev/null)" ]; then
+  echo "Please make sure there are no uncommitted changes."
+  exit 1
+fi
+
+# Get the last release branch.
+LAST_RELEASE_BRANCH=$(git branch --list 'release-*' | sort -V | grep -Eo 'release-[^ ]+' | tail -1)
+if [ -z "${LAST_RELEASE_BRANCH}" ]; then
+  echo "Unable to find the last release branch."
+  exit 1
+fi
```

Comment:
This could be a parameter instead. Sometimes we may need to merge previous release branch even after newer one exists.

Reply:
Done in https://github.com/grafana/mimir/pull/3234/commits/61a685125ac78a94bc4bc4af0c2540f4a644cc2d

WDYT?

Reply:
That works, thanks.